pandas/tests/indexes/test_multi.py

```diff
@@ -1704,6 +1704,11 @@ def test_to_hierarchical(self):
         tm.assert_index_equal(result, expected)
         assert result.names == index.names
 
+        # GH21613
+        # .to_hierarchical will be deprecated
+        with tm.assert_produces_warning(FutureWarning):
+            result = index.to_hierarchical(2)
```

Comment:
you don't need the result =

Reply:
Thanks - yes had missed it - will update together along with other comments